rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -2246,7 +2249,7 @@ message ClaimSwapNeuronsResponse {
   message SwapNeuron {
     NeuronId id = 1;
 
-    // The status of claiming of a requested Sale neuron.
+    // The status of claiming of a requested decentralization swap neuron.
```

Comment:
Nit.

Throughout the code, we typically just refer to "swap neurons" for brevity, although it's not wrong to call them "decentralization swap neuron".